src/nvim/api/win_config.c

```diff
@@ -207,9 +207,6 @@ Window nvim_open_win(Buffer buffer, Boolean enter, Dict(float_config) *config, E
 /// Configures window layout. Currently only for floating and external windows
 /// (including changing a split window to those layouts).
 ///
-/// When reconfiguring a floating window, absent option keys will not be
```

Comment:
is this no longer the case?

Reply:
it work likes before I think..only check when second reconfig missed relative field it can be before value.